scripts/generate_road_colours.py

```diff
@@ -54,9 +24,45 @@ def rgb_error(self):
 line_colour_infos["casing"] = ColourInfo(start_l = 50, end_l = 50, start_c = 70, end_c = 55)
 line_colour_infos["low-zoom-casing"] = ColourInfo(start_l = 50, end_l = 70, start_c = 50, end_c = 65)
 
+shield_colour_info = ColourInfo(start_l = 65, end_l = 80, start_c = 30, end_c = 30)
+colourless_roads = ["teriary"]
+
+
+
+
+#Start processing set data
+colour_divisions = len(road_classes) - 1
+hues = OrderedDict()
+
+# Because hue is a circle, it may be needed to add/subtract 360 to the min or
+# max when changing them
+delta_h = (max_h - min_h) / colour_divisions
+
+h = min_h
+for name in road_classes:
+  hues[name] = h
+  h = (h + delta_h) % 360
+
 # Colours for the MSS
 colours = OrderedDict()
 
+class Color:
```

Comment:
Classes are normally at the top of a file, or in their own file. If it was worth it, we'd do the latter, but that doesn't seem the best for what is a style project, not a code project.